lib/mongoid/copyable.rb

```diff
@@ -19,29 +19,49 @@ module Copyable
     def clone
       # @note This next line is here to address #2704, even though having an
       # _id and id field in the document would cause problems with Mongoid
-      # elsewhere.
+      # elsewhere. Note this is only done on the root document as we want
+      # to maintian the same _id on the embedded documents.
```

Comment:
maintain

Reply:
I assume maintaining _id on embedded documents through cloning is existing behavior, can it be added to docs as a note please?

Reply:
Yes that is existing behavior. I have added it to the docstring also, as explained above, we don't have docs on this either and I think the API docs is enough.